docs/index.rst

```diff
@@ -62,3 +59,22 @@ General Documentation
    getting_started
    tutorials/index
    api
+
+
+.. _astroplan_authors:
+
+Authors
+=======
+
+Maintainers
```

Comment:
My thinking is that this is a convenient way to both make it clear that this came out of a GSoC project, and make it clear who the maintainers our.  In practice I think *some* way of making it clear that they are maintainers is important, both for people to know who to talk to about it, and because that gives @bmorris3 and @jberlanga more meaningful titles to put on their CVs ("maintainer of astroplan" or "lead developer" or something along those lines is much better than "wrote some code for", even if both are true).

But it's easy enough to combine them together either now or in the future - I just thought this was convenient for now so that we don't have to figure out some other place to say this.